OTA-1743: Info about primary and secondary ECUs

merltron opened the pull request:
Adding extra conceptual info about primary and secondary ECUs according to the Uptane spec. Also linked to it from new topic about simulating devices.

Commit:
OTA-1743: Info about primary and secondary ECUs

Adding extra conceptual info about primary and secondary ECUs
according to the Update spec. Also luiinked to it from new topic
about simulating devices.

Signed-off-by: Merlin Carter <merlin.carter@here.com>

merltron commented:
@tkfu  This isn't directly related to the PR, but had misgivings about using the term "fake" in these instructions, since the word "fake" has a somewhat negative connotation (especially this year).  I wanted to reword to "simulate devices on your workstation" and refer to "simulated devices" instead of "fake" ones but didn't want to do it as part of this PR.

pattivacek reviewed (commented):
> "simulated devices" instead of "fake" ones

After considering your argument, I agree. I'd prefer "simulated" or "virtual" over "fake".

Also, you need to sign your comment. The link explains what to do but I can help if you have questions.

tkfu commented on _posts/2018-01-10-ats-garage-security-with-uptane.adoc:
```diff
@@ -25,3 +25,12 @@ The second comes from the *Director*, which controls what updates (selected from
+
+== Primary and secondary ECUs 
+
+anchor:primary-vs-secondary[]
```
A minor nit-pick in three parts:

1. The anchor, if you're going to use one, ought to be attached to the section title, not the first paragraph
1. It would be more idiomatic in asciidoc to do it in one of these two ways:

```
== Primary and secondary ECUs [[primary-vs-secondary]]

[#primary-vs-secondary]
== Primary and secondary ECUs
```

However, you can also do it without even having to define an ID/anchor link--just use the slugified form: `link:../concepts/ats-garage-security-with-uptane.html#primary-and-secondary-ecus`

tkfu reviewed (changes requested):
Nice! I like it. Also, I agree with you about the use of the work "fake", and I see no real problem in lumping that change in with this PR.

Commit:
Fix anchor and terminology

Reworded terminology from "fake" to "simulated".
Improved subheader link and removed anchor.
Reordered nav so that provisioning is 2nd
and device simulation comes directly after
image builds.

Signed-off-by: Merlin Carter <merlin.carter@here.com>

merltron commented:
@tkfu FYI: I responded to your change requests and update the files.

tkfu commented:
Great! Now you just need to [sign off on the commits](https://github.com/advancedtelematic/ats-garage-docs/pull/67/checks?check_run_id=37916776), thanks to the new DCO bot.

The instructions in the link above will tell you how to fix this PR, but in future you can just add a `-s` when you initially commit--i.e., if you would normally do `git commit -am 'This is my commit message'`, the only thing you need to change is to make is `git commit -asm 'This is my commit message'`.

merltron commented:
Ok, signed off my comments, merging now
